src/bin/collapse-bpftrace.rs

```diff
@@ -0,0 +1,36 @@
+use std::fs::File;
+use std::io;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+use inferno::collapse::bpftrace;
+
+const READER_CAPACITY: usize = 128 * 1024;
+
+#[derive(Debug, StructOpt)]
+#[structopt(name = "inferno-collapse-bpftrace", author = "")]
+struct Opt {
+    /// bpftrace script output file, or STDIN if not specified
+    infile: Option<PathBuf>,
+}
+
+fn main() -> io::Result<()> {
```

Comment:
I think it would be good if we could avoid all the duplication between this and the `collapse-perf.rs`..

Reply:
Well, the scripts take different command line arguments (if we want to remain faithful to the perl), but I'll think about how to address this as well since it is sad to duplicate similar code.

Reply:
I introduced an abstraction to get at this. Let me know if you think it was the wrong direction.